### PR TITLE
[build] Add doc build for new release branch

### PR DESCRIPTION
**Pull request by gyfora**

### CODE REVIEW

**Comment by gyfora** [comment]
cc @mbalassi @wangyang0918

File: .github/workflows/docs.yaml [hunk 1]
```diff
@@ -41,9 +42,11 @@ jobs:
           echo "flink_branch=${currentBranch}"
           echo "flink_branch=${currentBranch}" >> ${GITHUB_ENV}
           if [ "${currentBranch} = "main" ]; then
-            echo "flink_alias=release-1.0" >> ${GITHUB_ENV}
-          elif [ "${currentBranch} = "release-0.1" ]; then
+            echo "flink_alias=release-1.1" >> ${GITHUB_ENV}
```
Comment (wangyang0918) [inline .github/workflows/docs.yaml:45]: It seems that the step `upload documentation alias` not really works.

https://github.com/apache/flink-kubernetes-operator/runs/6516829906?check_suite_focus=true

Comment (gyfora) [reply .github/workflows/docs.yaml:45]: strange, maybe @mbalassi has a good idea what's going on here

**Comment by mbalassi** [comment]
Taking a look.

**Comment by mbalassi** [comment]
Indeed, if you check the [nigthlies root](https://nightlies.apache.org/flink/) the aliased docs are missing. Will play with it to see what is up.

**Comment by mbalassi** [comment]
I see, there is a typo in the script 😏 . Will fix later.

**Review by mbalassi — APPROVED** [review]
LGTM

**Comment by mbalassi** [comment]
Screenshot 2022-05-20 at 20 45 11

**Pull request merged**